contracts/NonfungibleTokenPositionDescriptor.sol

```diff
@@ -67,11 +59,33 @@ contract NonfungibleTokenPositionDescriptor is INonfungibleTokenPositionDescript
     }
 
     function flipRatio(address token0, address token1) public view returns (bool) {
-        return tokenRatioPriority[token0] > tokenRatioPriority[token1];
+        return tokenRatioPriority(token0) > tokenRatioPriority(token1);
     }
 
-    function updateTokenRatioPriority(TokenRatioOrderPriority calldata token) private {
-        tokenRatioPriority[token.token] = token.priority;
-        emit UpdateTokenRatioPriority(token.token, token.priority);
+    function tokenRatioPriority(address token) public view returns (int256) {
+        if (token == WETH9) {
+            return -100;
+        }
+        if (_chainid() == 1) {
+            if (token == 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48) {
+                // USDC
+                return 250;
```

Comment:
put these addresses in constants like in the test

also you can wrap the values in an enum for readability